apps/ccd/demo/base/logstash-secret.yaml

```diff
@@ -1,26 +1,26 @@
 apiVersion: v1
 data:
-    DATA_STORE_PASS: ENC[AES256_GCM,data:3PilSir6UlhpN1RRYihtSI2b7CAbXQMqu8ri9w==,iv:WfR60yI8KlRhC/4Gs+c+aGGj8CbCZ7amQkWYB8RqvwM=,tag:tf6ZFRr+hr7BP8YMqb2MwA==,type:str]
-    DATA_STORE_URL: ENC[AES256_GCM,data:gaVZxjnNI+7LWYuQPEcPmqqqkpV+FwhNQ7JSTC7bmZf4906m/BqWPgxL9trOyHYANwui96NWJEArDOonyKVcaW9Gei4FmQgYY+gtwgmobTtxbSxSBHcQFmZOSLRVt56NQoJHdBqslzOg6ztqIoxDzWHjUKq5OmlOQ97IYApavfH8hJ1eUX+xp3Oepbktj0MqsAo1nzNP4KqGG3ky/wPsZInnNRU=,iv:BVk0ZwFFOOWQ5RBBDrOsuvbk5DNKkLMjGdFJA6GhdI8=,tag:oO7UgbBc9DHcAytBsgi6rQ==,type:str]
-    DATA_STORE_USER: ENC[AES256_GCM,data:E8V5y/hJLM5u7ybB,iv:NZfld+WiI7p9CxhENte2u6Uqkf8BoBq2cq3o9ZJnhuY=,tag:K33gaOcMzSZqAhNrPPAJYQ==,type:str]
-    ES_HOSTS: ENC[AES256_GCM,data:UnpOCCsV02IuFTPB36eWvHSlf/bt6eEMi4ZLJQhDFGyGG5V45o+yDNCtwWOUFB0FzzkAd5+mcvQocxCA29QXfOZ856lkhT+3N40ZZg==,iv:stpkkq8EXhks8eizTh1BmmhBo4c0NqagGpK15DY856s=,tag:s+K+/wybW+xYg95bEBKQFA==,type:str]
+  DATA_STORE_INT_DEMO_PASS: ENC[AES256_GCM,data:a8PtGNPzDzN1mwxutqzE3PBnzJnYV3JIPibpaA==,iv:nI4A+aqO+POhNeS2+VlDkmnKUQ9odPGC1fg1NURpbEI=,tag:727lnjjoMdQUc/2w4yLV8g==,type:str]
+  DATA_STORE_INT_DEMO_URL: ENC[AES256_GCM,data:iMXb7/zjjO7cxKh9X6rbIDBLw3SB5zn3DvTGEdNxiJAUt4vsK8vGlgzP6UNwdQ0cOzdBeb9u2pkvhPJV4Ri5SCaX46vI3DFo6MUEQtZsSsXgarwk+03zGD2GcN5ovI0yms9gykWoknVs4p0ACMkydFRcMgHBmOP/y7DkI/9KIcI2k5gwC61XqemgMFVv/oY0c2JnHyxHxVi50V8+QFSiUrkHacRIjw9X,iv:6pP1BylutZjB4SUB1HnoIE3o4aADwCho5N+pjdNCqeQ=,tag:FWO7zW/Zkd1eOHXFBXZbYg==,type:str]
```

Comment:
All the secrets need to be in a new file or it will break demo and demo int needs to point to the new file